python/interpret_community/explanation/explanation.py

```diff
@@ -1348,6 +1357,9 @@ def _create_global_explanation(local_explanation=None, expected_values=None,
                                                        classification, explanation_id, **kwargs)
     DynamicGlobalExplanation = type(Dynamic.GLOBAL_EXPLANATION, tuple(mixins), {})
     global_explanation = DynamicGlobalExplanation(**kwargs)
+    if hasattr(local_explanation, ExplainParams.EVAL_DATA_VIZ):
+        setattr(global_explanation, ExplainParams.EVAL_DATA_VIZ,
+                getattr(local_explanation, ExplainParams.EVAL_DATA_VIZ))
```

Comment:
Same comment here - do this where we pull other components from the local explanation